src/test/java/com/parasoft/demoapp/graphql/ItemGraphQLDataFetcherTest.java

```diff
@@ -6,22 +6,31 @@
 import com.graphql.spring.boot.test.GraphQLTestError;
 import com.graphql.spring.boot.test.GraphQLTestTemplate;
 import com.parasoft.demoapp.controller.PageInfo;
+import com.parasoft.demoapp.dto.ItemsDTO;
+import com.parasoft.demoapp.messages.AssetMessages;
 import com.parasoft.demoapp.model.industry.ItemEntity;
+import com.parasoft.demoapp.model.industry.RegionType;
+import com.parasoft.demoapp.service.GlobalPreferencesService;
 import org.assertj.core.api.Condition;
 import org.assertj.core.data.Index;
+import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TestName;
 import org.junit.runner.RunWith;
```

Comment:
Please also add test cases:

- empty description
- inStock = null (we can set it to null in Literal mode in SOAtest)